lib/bootloader_setup.pm

```diff
@@ -212,6 +212,11 @@ sub specific_bootmenu_params {
         }
     }
 
+    if (check_var('ARCH', 'aarch64') && check_var('DISTRI', 'sle')) {
+        $args .= " plymouth=0 linuxrc.debug=trace";
+        record_soft_failure "bsc#1005313 - Unhandled level 1 translation fault";
+    }
```

Comment:
you sure can have a bootup scenario that stops at welcome but adds tons of debug options.